src/fides/api/models/sql_models.py

```diff
@@ -450,10 +507,19 @@ class PrivacyDeclaration(Base):
         nullable=False,
         index=True,
     )
-    system = relationship(System, back_populates="privacy_declarations")
+    system = relationship(
+        System, back_populates="privacy_declarations", lazy="selectin"
+    )
```

Comment:
Need to be careful of this change, it makes the system retrieval on privacy declarations eager

Reply:
Are you concerned about performance on this PR?

Reply:
Systems already feel very bogged down, at some point we may reach a tipping point where we need to rearchitect some of this -